nekoyume/Assets/_Scripts/UI/Module/GrindModule.cs

```diff
@@ -116,14 +114,15 @@ public void Show(bool reverseInventoryOrder = false)
                 OnUpdateInventory,
                 DimConditionPredicateList,
                 reverseInventoryOrder);
-            grindButton.Interactable = false;
+            UpdateScroll();
             UpdateStakingBonusObject(States.Instance.StakingLevel);
             crystalRewardText.text = string.Empty;
         }
 
         private void OnDisable()
         {
             _disposables.DisposeAllAndClear();
+            _selectedItemsForGrind.Clear();
```

Comment:
```suggestion
            foreach (var item in _selectedItemsForGrind.ToList())
            {
                item.GrindingCountEnabled.SetValueAndForceNotify(false);
            }

            _selectedItemsForGrind.Clear();
```

Reply:
that is just for clean scroll model, don't need to inventory. when show UI_Grind again, that will be call update `OnUpdateInventory` method.